wallet-api/src/Wallet/UTXO/Runtime.hs

```diff
@@ -38,6 +38,12 @@ import           Language.Plutus.Lift (LiftPlc (..), TypeablePlc (..))
 import           Wallet.UTXO.Types    (PubKey (..), Signature (..))
 import qualified Wallet.UTXO.Types    as UTXO
 
+-- Ignore newtype warnings related to `Oracle` and `Signed` because it causes
+-- problems with the plugin
+--
+-- TODO: Remove annotation once `newtype Oracle = (...)` works
```

Comment:
I think the `newtype` should work, you just have to use `Generic` deriving rather than `newtype` deriving - can you try that?

Reply:
It's a different error - caused by the plugin at compile time:

```
[5 of 7] Compiling Language.Plutus.Coordination.Contracts.Future ( src\Language\Plutus\Coordination\Contracts\Future.hs, 
.stack-work\dist\5c8418a7\build\Language\Plutus\Coordination\Contracts\Future.o ) [Wallet.UTXO.Runtime changed]

Error: Unsupported: Coercion
       N:OracleValue[0] <Value>_R
Context: Converting expr: ov_a17cH
                          `cast` (N:OracleValue[0] <Value>_R
                                  :: Coercible (OracleValue Value) (Signed (Height, Value)))
Context: Converting expr: case ov_a17cH
                               `cast` (N:OracleValue[0] <Value>_R
                                       :: Coercible (OracleValue Value) (Signed (Height, Value)))
                          of
                          { Signed ds_d182Y ->
                          case ds_d182Y of { (pk_a17cF, t_a17cG) ->
                          case eqPk_a17cd pk_a17cF ds_d184c of {
                            False -> error @ (Height, Value) ();
                            True -> t_a17cG
                          }
                          }
}
```

Full context [here](https://gist.github.com/j-mueller/a7ca5f732c61b5f62ae83f14686a1e2a)

It only happens with `newtype OracleValue a = OracleValue (Signed (Height, a))`, not any of the other newtypes. 

(I'll create a test case/YT issue if that's a new one)

Reply:
Yeah, please create a ticket. Although I can't promise we'll support it - looks like more of GHC's coercion magic for newtypes.

Reply:
https://iohk.myjetbrains.com/youtrack/issue/CGP-414